lib/process.dart

```diff
@@ -3,4 +3,5 @@
 // BSD-style license that can be found in the LICENSE file.
 
 export 'src/interface/local_process_manager.dart';
+export 'src/interface/process_exception.dart';
```

Comment:
This package currently doesn't export anything from `dart:io` or conflicting with `dart:io`.  There's no need to do so to satisfy #7.

I'm open to exporting more stuff, but if we do so, we should probably just export `dart:io` classes directly rather than re-defining them.  e.g.:

```
export `dart:io`
    show
        Process,
        ProcessException,
        ProcessResult,
        ProcessSignal,
        ProcessStartMode;
```

Reply:
Exporting the `dart:io` classes is clearly the best option as of today.